src/model.d.ts

```diff
@@ -2213,7 +2213,7 @@ export abstract class Model<TModelAttributes extends {} = any, TCreationAttribut
     values: {
         [key in keyof Attributes<M>]?: Attributes<M>[key] | Fn | Col | Literal;
     },
-    options: UpdateOptions<Attributes<M>> & { returning: true }
+    options: UpdateOptions<Attributes<M>> & { returning: true | (keyof Attributes<M>)[] }
```

Comment:
doing this is a bit more verbose but will avoid having to update this signature if `UpdateOptions` is modified

```suggestion
    options: Omit<UpdateOptions<Attributes<M>>, 'returning'> 
      & { returning: Exclude<UpdateOptions<Attributes<M>>['returning'], undefined | false> }
```